### PR TITLE
Added missing fields for detections

### PR DESCRIPTION
**Pull request by ashwin-patil**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/SecurityEvent/TimeSeriesAnomaly-ProcessExecutions.txt [hunk 1]
```diff
@@ -6,6 +6,16 @@
...
 //
+// Severity: Medium
+//
+// QueryFrequency: 1h
+//
+// QueryPeriod: 30d
+//
+// AlertTriggerOperator: gt
+//
+// AlertTriggerThreshold: 0
```
Comment (shainw) [inline Detections/SecurityEvent/TimeSeriesAnomaly-ProcessExecutions.txt]: I think you want this to be 5, as that is your value in the script - let TotalEventsThreshold = 5;

File: Detections/CommonSecurityLog/TimeSeriesAnomaly-MultiVendor_NetworkTraffic.txt [hunk 2]
```diff
@@ -6,6 +6,16 @@
...
 //
+// Severity: Medium
+//
+// QueryFrequency: 1h
+//
+// QueryPeriod: 30d
+//
+// AlertTriggerOperator: gt
+//
+// AlertTriggerThreshold: 0
```
Comment (shainw) [inline Detections/CommonSecurityLog/TimeSeriesAnomaly-MultiVendor_NetworkTraffic.txt]: Maybe this should be 3 as that is what you are using in the series_decompose_anomalies function.

Comment (ashwin-patil) [reply Detections/CommonSecurityLog/TimeSeriesAnomaly-MultiVendor_NetworkTraffic.txt]: Agreed per discussion to match scorethreshold with alerttriggerthreshold. 
Submitted changes to both files where scorethreshold is used.

File: Detections/SecurityEvent/TimeSeriesAnomaly-ProcessExecutions.txt (hunk 1 above)
Comment (ashwin-patil) [reply Detections/SecurityEvent/TimeSeriesAnomaly-ProcessExecutions.txt]: event threshold 5 used here to exclude events with low number of process executions count to reduce false positives however it is ambiguous in this case. Per discussion, we can wait to see how alerts are going to be configured using threshold to make changes here.

**Pull request merged**